.rubocop.yml

```diff
@@ -121,3 +122,21 @@ Style/TrailingWhitespace:
 Style/VariableName:
   Severity: fatal
   Enabled: true
+
+Metrics/LineLength:
+  Description: 'Limit lines to 120 characters.'
```

Comment:
80 characters is also the standard in Go, so helps to be consistent. We have seen problems with heavily namespaced classes though...